.claude/commands/generate-prp.md

```
@@ -1,69 +1,636 @@
-*** CRITICAL AFTER YOU ARE DONE RESEARCHING AND EXPLORING THE CODEBASE BEFORE YOU START WRITING THE PRP ***
+**NO manual steps required!**
+
+---
 
-*** ULTRATHINK ABOUT THE PRP AND PLAN YOUR APPROACH THEN START WRITING THE PRP ***
+## 📚 References & Best Practices
 
-## Output
-Save as: `PRPs/{feature-name}.md`
+### Anthropic Documentation:
+- **Subagents**: https://docs.claude.com/en/docs/claude-code/sub-agents
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)